docs/contributing.rst

```diff
@@ -115,6 +115,7 @@ To run the integration tests for the first time, use:
 
     export AIRFLOW_HOME=`pwd`
     export AIRFLOW_CONN_AIRFLOW_DB=postgres://postgres:postgres@0.0.0.0:5432/postgres
```

Comment:
Thank you so much for updating the contributing guidelines, @t0momi219 !
This one is no longer needed -`AIRFLOW_CONN_EXAMPLE_CONN` replaced `AIRFLOW_CONN_AIRFLOW_DB`, could you update this? We'll merge once it is done 🎉

Reply:
Thank you for your review!
It seems that the missing variable caused an error in my local environment, but I believe it was an issue on my side.
I’ve removed the variable.